indexing-solr/src/main/java/edu/unc/lib/boxc/indexing/solr/filter/SetContentStatusFilter.java

```diff
@@ -53,6 +53,9 @@ private List<String> determineContentStatus(DocumentIndexingPackage dip)
             if (parentResc.hasProperty(Cdr.primaryObject, resc)) {
                 status.add(FacetConstants.IS_PRIMARY_OBJECT);
             }
+            if (parentResc.hasProperty(Cdr.useAsThumbnail, resc)) {
```

Comment:
It'd be good to add tests to `SetContentStatusFilterTest` for the new status values being added